app/imports/components/posts/NewPost.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import { addPost } from '/imports/api/photosMethods.js';
```

Comment:
`NewPost` is a container `addPost` is a method. imo its alright. a component is not tied to 1 method